cmd/common/common.go

```diff
@@ -264,9 +282,9 @@ func WriteFileOrStdout(data []byte, output string) error {
 	return nil
 }
 
-// HandleError will generically handle an error by logging its contents
+// LogErrorAndExit will generically handle an error by logging its contents
 // and exiting with a return code of 1.
-func HandleError(err error) {
+func LogErrorAndExit(err error) {
```

Comment:
`LogErrorAndExit` is a confusing name - leads me to believe that an error is not nil

Maybe call it `LogAndExitIfError` or something that indicates there is a nil error check?

Reply:
Maybe I should just rename it back to `HandleError`? I was thinking `LogErrorAndExit` might be a better name but I see your point

Reply:
I do like knowing that it handles the error in a fatal manner, so having `Exit` in there is good...  maybe `ExitIfError`?